src/include/txn_inline.h

```diff
@@ -940,7 +993,10 @@ __wt_txn_upd_visible_type(WT_SESSION_IMPL *session, WT_UPDATE *upd)
               upd->type == WT_UPDATE_STANDARD))
             return (WT_VISIBLE_TRUE);
 
-        upd_visible = __wt_txn_visible(session, upd->txnid, upd->start_ts, upd->durable_ts);
+        // FIXME-WT-11469
+        WT_ASSERT(session, __wt_txn_upd_get_durable(session, upd, &durable_ts) == 0);
```

Comment:
If anyone had input for this would be appreciated. This function has a return type of `WT_VISIBLE_TYPE` which doesn't allow use of a `WT_RET` here, as such I've currently used a WT_ASSERT.